README.md

```diff
@@ -330,7 +330,42 @@ ctest -C RelWithDebInfo --parallel 8
 On Mac OSX using Xcode
 ======================
 
-#### Get the dependencies
+#### For Mac OSX 10.10 Yosemite and OS X 10.11 El Capitan
+Get **Xcode** from the App store. Open **Xcode** and *Agree* to license agreement. To *Agree* to to the license agreement, you may need to type in **Terminal**:
+```shell 
+sudo xcodebuild -license
+``` 
+
+Then, in **Terminal**, copy and paste commands below, line by line, one at a time. Be sure the output doesn't contain errors.
+```shell
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+brew install cmake swig
+brew cask install java
+git clone https://github.com/opensim-org/opensim-core.git
+mkdir opensim_dependencies_build
+cd opensim_dependencies_build
+cmake ../opensim-core/dependencies \
+      -DCMAKE_INSTALL_PREFIX="~/opensim_dependencies_install" \
+      -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Comment:
The comment about "line by line" above may be confusing when users get to this part: this one command takes up 3 lines (with `\` to escape the newline), and it may be better to copy this whole multi-line command at once instead of line by line. @camilletown what did you do when you used these commands? Did you copy each of these 3 lines one at a time? If that worked fine then we can leave in "line by line."